reference/entry.json

```diff
@@ -2698,6 +2698,85 @@
         ]
       }
     },
+    "/organizers/{organizerId}/name": {
+      "parameters": [
+        {
+          "$ref": "#/components/parameters/organizerId"
+        }
+      ],
+      "put": {
+        "summary": "Update name nl",
+        "description": "Updates the name of the organizer with the given `organizerId`. The new name of the request body will be set on the organizer with default language `nl`. The new name needs to be at least one character long.",
```

Comment:
Since this is a deprecated endpoint it would be good to add a warning to the description with a link to the endpoint that should be used instead

Reply:
Is it possible to add a link to the endpoint that should be used? I checked https://meta.stoplight.io/docs/studio/ZG9jOjg0-stoplight-flavored-markdown-smd but did not find it.

Reply:
It used to be possible in the old design but it's very error-prone. So maybe just mention the name of the operation in the sidebar and/or something like "use `PUT /organizers/{organizerId}/name/{language}` instead".